lib/cache.c

```diff
@@ -1,21 +1,25 @@
-/*****************************************************************************/
-/*  cache.c - contains the cache routines                                    */
-/*  Copyright (C) 1998-2007 Brian Masney <masneyb@gftp.org>                  */
-/*                                                                           */
-/*  This program is free software; you can redistribute it and/or modify     */
-/*  it under the terms of the GNU General Public License as published by     */
-/*  the Free Software Foundation; either version 2 of the License, or        */
-/*  (at your option) any later version.                                      */
-/*                                                                           */
-/*  This program is distributed in the hope that it will be useful,          */
-/*  but WITHOUT ANY WARRANTY; without even the implied warranty of           */
-/*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the            */
-/*  GNU General Public License for more details.                             */
-/*                                                                           */
-/*  You should have received a copy of the GNU General Public License        */
-/*  along with this program; if not, write to the Free Software              */
-/*  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111 USA      */
-/*****************************************************************************/
+/***********************************************************************************/
+/*  cache.c - contains the cache routines                                          */
+/*  Copyright (C) 1998-2021 Brian Masney <masneyb@gftp.org>                        */
```

Comment:
Per the issue, I'm ok with this for my contributions. This needs one minor change though. The old line had 1998-2007 for my changes to this file, but this changed to include 2021. I think you should leave my copyright lines alone since I haven't modified these files in years. You can add a new line if you'd like:

```
Copyright (C) 1998-2007 Brian Masney <masneyb@gftp.org> 
Copyright (C) 2020-2021 YOUR NAME <...>
```